03-first-aws-environment/bin/uniquify.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
I understand why you have this script, but seems like we're doing something wrong if we need it. Maybe something we should do differently in the component.

Reply:
@osterman Yeah, it's not the best situation. What I want is to be able to use an unchanged version of the `tfstate-backend` component, but unfortunately that produces consistent bucket names. If we want to introduce `random_pet` into both the `tfstate-backend` and `static-site` components to create unique names then we can do so, but then we'll require the tutorial followers to have to do manual updates to some of the stack files. Maybe that is fine? But it seemed undesirable.

Reply:
What is the `tfstate-backend` writes the `stacks/catalog/backend.yaml` file with all the settings?

Reply:
I'm okay with this script approach. Let's ship this sooner. 

Can we just rename this script to `random-pet.sh`. The `uniquify` script doesn't have such a nice ring to it =)